src/main/java/kernitus/plugin/OldCombatMechanics/utilities/packet/PacketSender.java

```diff
@@ -6,39 +6,25 @@
 
 import java.lang.reflect.Field;
 import java.lang.reflect.Method;
-import java.util.Objects;
 
 /**
  * A Packet sender
  */
-class PacketSender {
-
-    private static Class<?> CRAFT_PLAYER;
-    private static Class<?> PLAYER_CONNECTION;
-    private static Class<?> ENTITY_PLAYER;
-
-    private static Method GET_HANDLE;
-    private static Method SEND_PACKET;
-    private static Field PLAYER_CONNECTION_FIELD;
-
-    private static boolean isSetup = false;
-
+public class PacketSender {
     private static final PacketSender instance = new PacketSender();
 
-    static{
-        try{
```

Comment:
why remove the `try`? i'd rather avoid throwing exceptions in static init blocks (or constructors) since a single module failing shouldn't bring down the whole plugin

Reply:
We'd probably need some more heavy-duty isolation then. I can re-add the try but the effects might still appear in all modules related to this and will happen every time they try to use it.

Maybe we'd some slightly more standardizes "Initializable" that can fail and force clients to check for it before they use it? That sounds a bit annoying to use though.

Reply:
If the `PacketSender` fails to load all `send` Operations will fail and the `PacketInjector` will fail to `attach`. Any of these Operations will fail with a try as well. Therefore it is probably not really helpful to have the try there.

If we want OCM to be resilient to module failures we'd need some isolation which doesn't rely on knowing the whole dependency graph of everything that can throw an exception. We could pass a lambda creating a new module instance and execute all the actual instantiations and all enable/disable listeners in quite broad try-catch blocks.